src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -379,8 +374,7 @@ public void equals() {
         assertFalse(modelManager.equals(new ModelManager(cliTutors, userPrefs)));
 
         // different match list
-        ArrayList<Tag> tagList = new ArrayList<>();
-        tagList.addAll(DANIEL.getTags());
+        ArrayList<Tag> tagList = new ArrayList<>(DANIEL.getTags());
```

Comment:
Extremely minor point:
```suggestion
        List<Tag> tagList = new ArrayList<>(DANIEL.getTags());
```